app/src/main/java/com/anytypeio/anytype/ui/primitives/ObjectTypeFragment.kt

```diff
@@ -0,0 +1,321 @@
+                    }
+
+                    OpenObjectNavigation.NonValidObject -> {
+                        toast(getString(R.string.error_non_valid_object))
+                    }
+
+                    is OpenObjectNavigation.OpenDateObject -> {
+                        runCatching {
+                            findNavController().navigate(
+                                R.id.dateObjectScreen,
+                                DateObjectFragment.args(
+                                    objectId = nav.target,
+                                    space = nav.space
+                                )
+                            )
+                        }.onFailure {
+                            Timber.e(it, "Failed to navigate to date object screen")
+                        }
+                    }
+
+                    is OpenObjectNavigation.UnexpectedLayoutError -> {
+                        toast(getString(R.string.error_unexpected_layout))
+                    }
+
+                    else -> {
+                        // Do nothing.
+                    }
+                }
+            }
+        }
+    }
+
+    @OptIn(ExperimentalMaterial3Api::class)
+    @Composable
+    private fun ErrorScreen() {
+        val errorStateScreen = vm.errorState.collectAsStateWithLifecycle().value
+        if (errorStateScreen is UiErrorState.Show) {
+            when (val r = errorStateScreen.reason) {
+                is UiErrorState.Reason.ErrorGettingObjects -> {
+                    BaseAlertDialog(
+                        dialogText = "${stringResource(R.string.object_type_open_type_error)}:\n${r.msg}",
+                        buttonText = stringResource(id = R.string.membership_error_button_text_dismiss),
+                        onButtonClick = vm::closeObject,
+                        onDismissRequest = vm::closeObject
+                    )
+                }
+                is UiErrorState.Reason.Other -> {
+                    BaseAlertDialog(
+                        dialogText = r.msg,
+                        buttonText = stringResource(id = R.string.membership_error_button_text_dismiss),
+                        onButtonClick = vm::hideError,
+                        onDismissRequest = vm::hideError
+                    )
+                }
+            }
+        }
+        when (val state = errorStateScreen) {
+            UiErrorState.Hidden -> {
+
+            }
+
+            is UiErrorState.Show -> {
+
+            }
+        }
+    }
+
+    override fun injectDependencies() {
+        val params = ObjectTypeVmParams(
+            spaceId = SpaceId(space),
+            objectId = objectId,
+            withSubscriptions = true,
+            showHiddenFields = true
+        )
+        componentManager().objectTypeComponent.get(params).inject(this)
+    }
+
+    override fun releaseDependencies() {
+        componentManager().objectTypeComponent.release()
+    }
+
+    override fun onApplyWindowRootInsets(view: View) {
```

Comment:
do not override this method here if you don't use window insets modifiers in the compose code related to this screen